From 705845a613139dd1d02a587478d8b7e93f16eecf Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Wed, 8 May 2024 22:39:40 +0200
Subject: [PATCH] include: Sync nf_log.h with kernel headers

JIRA: https://issues.redhat.com/browse/RHEL-28515
Upstream Status: libnftnl commit 9da7658c6e25b02f7eeef936835469f4174cbfec

commit 9da7658c6e25b02f7eeef936835469f4174cbfec
Author: Phil Sutter <phil@nwl.cc>
Date:   Fri Dec 15 16:15:35 2023 +0100

    include: Sync nf_log.h with kernel headers

    Next patch needs NF_LOG_PREFIXLEN define.

    Signed-off-by: Phil Sutter <phil@nwl.cc>

Signed-off-by: Phil Sutter <psutter@redhat.com>
---
 include/linux/netfilter/nf_log.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/linux/netfilter/nf_log.h b/include/linux/netfilter/nf_log.h
index 8be21e0..2ae0093 100644
--- a/include/linux/netfilter/nf_log.h
+++ b/include/linux/netfilter/nf_log.h
@@ -1,3 +1,4 @@
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _NETFILTER_NF_LOG_H
 #define _NETFILTER_NF_LOG_H
 
@@ -9,4 +10,6 @@
 #define NF_LOG_MACDECODE	0x20	/* Decode MAC header */
 #define NF_LOG_MASK		0x2f
 
+#define NF_LOG_PREFIXLEN	128
+
 #endif /* _NETFILTER_NF_LOG_H */
